### ISSUE TITLE
Loss function different than in original paper

### ISSUE THREAD

**Issue by zlenyk** [jramapuram/BYOL#5]
Hi! Could you explain why your loss function is -2 * torch.sum(x * y, dim=-1) / (norm_x * norm_y), but the original paper mentions "2 - 2*(...)" ? 
Is this expected?

Thank you
